aptos-move/move-examples/token_objects/ambassador/move/sources/ambassador.move

```diff
@@ -0,0 +1,265 @@
+/// This module is used to create ambassador tokens. Ambassador tokens are
+/// soulbound tokens that are minted by the creator of a collection.
```

Comment:
could we add more description here on levels, management, events, etc?

Reply:
Sure, I added some more description here and throughout the module.